README.md

```diff
@@ -68,9 +68,14 @@ To run the `classify` command, use `poetry run`:
 poetry run classify --config config.json
 ```
 
-If you re-run the classifier, it will look in the cache for each case / prompt combination and return a cached result if it exists, rather than going to the LLM. For now, the only way to stop caching is to delete the cache file or directory for a prompt or case.
+If you re-run the classifier, it will look in the cache for each case / prompt
+combination and return a cached result if it exists, rather than going to the
+LLM. For now, the only way to stop caching is to delete the cache file or
+directory for a prompt or case.
 
```

Comment:
Now we have `no_cache` flag, should this sentence be changed?

Reply:
Done (and made sure it's available on the command line as well